src/main/scala/io/iohk/ethereum/vm/ProgramError.scala

```diff
@@ -9,3 +9,5 @@ case class  InvalidOpCode(code: Byte) extends ProgramError
 sealed trait StackError extends ProgramError
 case object StackOverflow extends StackError
 case object StackUnderflow extends StackError
+
+sealed trait ContractCreationError extends ProgramError
```

Comment:
A sealed trait without implementation? I understand it's an unfinished business but are you sure we need such an error? See YP 9.4.2 and the definition of `x` in `CREATE` instruction description.